include/z64actor.h

```diff
@@ -355,6 +355,103 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    /* 0x00 */ NAVI_ENEMY_DEFAULT,
+    /* 0x01 */ NAVI_ENEMY_GOHMA,
+    /* 0x02 */ NAVI_ENEMY_GOHMA_EGG,
+    /* 0x03 */ NAVI_ENEMY_GOHMA_LARVA,
+    /* 0x04 */ NAVI_ENEMY_SKULLTULA,
+    /* 0x05 */ NAVI_ENEMY_BIG_SKULLTULA,
+    /* 0x06 */ NAVI_ENEMY_TAILPASARAN,
+    /* 0x07 */ NAVI_ENEMY_DEKU_BABA,
+    /* 0x08 */ NAVI_ENEMY_BIG_DEKU_BABA,
+    /* 0x09 */ NAVI_ENEMY_WITHERED_DEKU_BABA,
+    /* 0x0A */ NAVI_ENEMY_DEKU_SCRUB,
+    /* 0x0B */ NAVI_ENEMY_UNUSED11,
+    /* 0x0C */ NAVI_ENEMY_KING_DODONGO,
+    /* 0x0D */ NAVI_ENEMY_DODONGO,
+    /* 0x0E */ NAVI_ENEMY_BABY_DODONGO,
+    /* 0x0F */ NAVI_ENEMY_LIZALFOS,
+    /* 0x10 */ NAVI_ENEMY_DINOLFOS,
+    /* 0x11 */ NAVI_ENEMY_FIRE_KEESE,
+    /* 0x12 */ NAVI_ENEMY_KEESE,
+    /* 0x13 */ NAVI_ENEMY_ARMOS,
+    /* 0x14 */ NAVI_ENEMY_BARINADE,
+    /* 0x15 */ NAVI_ENEMY_PARASITIC_TENTACLE,
+    /* 0x16 */ NAVI_ENEMY_SHABOM,
+    /* 0x17 */ NAVI_ENEMY_BIRI,
+    /* 0x18 */ NAVI_ENEMY_BARI,
+    /* 0x19 */ NAVI_ENEMY_STINGER,
+    /* 0x1A */ NAVI_ENEMY_PHANTOM_GANON_2,
+    /* 0x1B */ NAVI_ENEMY_STALFOS,
+    /* 0x1C */ NAVI_ENEMY_BLUE_BUBBLE,
+    /* 0x1D */ NAVI_ENEMY_WHITE_BUBBLE,
+    /* 0x1E */ NAVI_ENEMY_GREEN_BUBBLE,
+    /* 0x1F */ NAVI_ENEMY_SKULLWALLTULA,
+    /* 0x20 */ NAVI_ENEMY_GOLD_SKULLTULA,
+    /* 0x21 */ NAVI_ENEMY_VOLVAGIA,
+    /* 0x22 */ NAVI_ENEMY_FLARE_DANCER,
+    /* 0x23 */ NAVI_ENEMY_TORCH_SLUG,
+    /* 0x24 */ NAVI_ENEMY_RED_BUBBLE,
+    /* 0x25 */ NAVI_ENEMY_MORPHA,
+    /* 0x26 */ NAVI_ENEMY_DARK_LINK,
+    /* 0x27 */ NAVI_ENEMY_SHELL_BLADE,
+    /* 0x28 */ NAVI_ENEMY_SPIKE,
+    /* 0x29 */ NAVI_ENEMY_BONGO_BONGO,
+    /* 0x2A */ NAVI_ENEMY_REDEAD,
+    /* 0x2B */ NAVI_ENEMY_PHANTOM_GANON_1,
+    /* 0x2C */ NAVI_ENEMY_UNUSED2C,
+    /* 0x2D */ NAVI_ENEMY_GIBDO,
+    /* 0x2E */ NAVI_ENEMY_DEAD_HANDS_HAND,
+    /* 0x2F */ NAVI_ENEMY_DEAD_HAND,
+    /* 0x30 */ NAVI_ENEMY_WALLMASTER,
+    /* 0x31 */ NAVI_ENEMY_FLOORMASTER,
+    /* 0x32 */ NAVI_ENEMY_TWINROVA_KOUME,
+    /* 0x33 */ NAVI_ENEMY_TWINROVA_KOTAKE,
+    /* 0x34 */ NAVI_ENEMY_IRON_KNUCKLE_MINIBOSS,
```

Comment:
I think this is nabooru-knuckle maybe? all iron knuckles qualify as minibosses for me, so if this is indeed nabooru-knuckle maybe `NAVI_ENEMY_IRON_KNUCKLE_NABOORU`

Reply:
It is nabooru, I was on the fence about calling her that cause theres nothing in the game that really calls her that. 
I see it as more of a miniboss than the others, but if its the only distinguishing factor than so be it

Reply:
Technically they do all use the miniboss music. I think the common name is "Nabooru Knuckle"?

Reply:
thats the colloquial name for sure, but the navi enemy text at least doesnt say that. I guess cause its supposed to be a suprise for when you beat it

Reply:
Yeah, definitely, but it's certainly not the only place this PR deviates from the boxes for clarity. You could even change the Phantom Ganon ones to refer to horsed/unhorsed or something.

Reply:
for phantom ganon i was just going to add "phase" to the number.

but yea, nabooru is fine. I would have preferred miniboss but if that is unclear then oh well